src/containers/shared/components/Transaction/AMMClawback/test/AMMClawbackSimple.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { expectSimpleRowNotToExist, expectSimpleRowText } from '../../test'
+
+import { createSimpleWrapperFactory } from '../../test/createWrapperFactory'
+import { Simple } from '../Simple'
+import mockAMMClawbackNoFlag from './mock_data/withoutFlag.json'
+import mockAMMClawbackWithAmount from './mock_data/withAmount.json'
+import mockAMMClawbackWithFlag from './mock_data/withFlag.json'
+
+const createWrapper = createSimpleWrapperFactory(Simple)
+
+describe('AMMClawback: Simple', () => {
+  it('renders without tfClawTwoAssets flag, only one asset should be clawed back', () => {
+    const wrapper = createWrapper(mockAMMClawbackNoFlag)
+    expectSimpleRowText(
```

Comment:
I'm trying to understand the difference between this file and`src/containers/shared/components/Transaction/AMMClawback/test/AMMClawbackTableDetail.test.tsx`. Both of them are testing the three different possibilities for the `AMMClawback` transaction. How are they different?

Reply:
`TableDetail` vs `Simple`